APIC.md

```diff
@@ -90,9 +111,55 @@ So basically if we want to read/write a register of the IOAPIC we need to:
 1. write the register index in the IOREGSEL register
 2. read/write the content of the register selected in IOWIN register
 
-### IOREDTBL
+### Interrupt source overrides
+They contains differences between the IA-PC standard and the dual 8250 interrupt definitions. The isa interrupts should be identity mapped into the first IO-APIC sources, but most of the time there will be at least one exception. This table contains those exceptions. 
+An example is the PIT Timer is connected to ISA IRQ 0, but when apic is enabled it is connected to the IO-APIC interrupt input pin 2, so in this case we need an interrupt source override where the Source entry (bus source) is 0 and the global system interrupt is 2
+The values stored in the IO Apic Interrupt source overrides in the MADT are:
 
-TODO
+| Offset | Length | Description                  |
+|--------|--------|------------------------------|
+| 2      | 1      | bus source (it should be 0)  |
+| 3      | 1      | irq source                   |
+| 4      | 4      | Global System Interrupt      |
+| 8      | 2      | Flags                        |
+
+* Bus source usually is constant and is 0 (is the ISA irq source)
```

Comment:
bus source is actually a reserved field in acpi v2+, so it can also be safely ignored.